NugetLightRequest.cs

```diff
@@ -58,9 +58,6 @@ public abstract class NuGetRequest : Request {
         private HttpClient _httpClient;
         private HttpClient _httpClientWithoutAcceptHeader;
         private bool? _isCalledFromPowerShellGet;
-        private static readonly string _dollarPSHome = "$PSHome";
-        private static bool _isWindow;
-        private static bool _psHomePathInitialized;
```

Comment:
Bryan, I never like this. can you take this chance to fix them if possible?  Here is the story:

Some code in the PackageMangement are shared with its providers. This thing happened when we moved from source depot to github.
 The above 3 lines I think are used once we integrate with PackageManagent code. SO need to keep it. Very nasty. 

Shared code:
Microsoft.PackageManagement/providers/inbox/Common   === I think nobody else other than NuGet provider is using it. 
and may be Microsoft.PackageManagement\Utility   --- shared.

I found due to this shared code problem, NuGet is tied up wiht PackageManagement which is fine from design perspective. But it's awkward and hard for NuGet code if you just want to fix some bugs in it.